README.md

```diff
@@ -6,6 +6,31 @@
   <h4 align="center">A Flutter-based Android App</h4>
 </p>
 
+The app is a complete free handbook of Basics of C Programming language which covers important topics, notes & programs on the course. Download the App as a reference material for Computer Science & software Engineering programs & Degree courses.
+
+C programming is a powerful general-purpose language. If you are new to programming then C Programming is the best language to start your programming journey. In practical C programming is used in Embedded stuff, Systems Programming.
+
+## Features
+* [x] Android (iOS Coming Soon)
+
+  * [x] Contains all basic concepts of 'C' programming language (C Language).
+  * [x] C programs with comments for better understanding (300+ programs)
```

Comment:
>C programs with comments for better understanding (300+ programs)

we do not have 300+ programs in our app right now.
maybe we will have in future.
According to me, its not right to include this.